src/components/datagrid/body/data_grid_cell_popover.tsx

```diff
@@ -44,6 +44,9 @@ export function EuiDataGridCellPopover({
       zIndex={8001}
       display="block"
       closePopover={closePopover}
+      panelProps={{
+        'data-test-subj': 'euiDataGridExpansionPanel',
```

Comment:
[Super optional] We seem to call this an `expansion popover` in https://elastic.github.io/eui/#/tabular-content/data-grid-schemas-and-popovers - not sure if it's worth sticking to that that for consistency

```suggestion
        'data-test-subj': 'euiDataGridExpansionPopover',
```